addons/disable-sprite-wobble/addon.json

```diff
@@ -9,17 +9,43 @@
     {
       "name": "CST1229",
       "link": "https://github.com/cst1229/"
+    },
+    {
+      "name": "DNin01",
+      "link": "https://github.com/DNin01"
     }
   ],
   "userstyles": [
     {
       "url": "userstyle.css",
       "matches": ["projects"]
+    },
+    {
+      "url": "disable-wobble.css",
+      "matches": ["projects"],
+      "if": {
+        "settings": {
+          "disable-wobble": true
+        }
+      }
+    }
+  ],
+  "settings": [
+    {
+      "id": "disable-wobble",
+      "name": "Disable sprite wobble",
```

Comment:
I wonder how we could name this setting so that it doesn't feel "mandatory" to enable it. Right now, I believe some users might believe the addon does nothing if this isn't enabled.

Reply:
We can add a `Fix glitch` setting that defaults to true (a bit stupid, but it would work).
Instead we could also rename this setting to something like `Also disable sprite wobble` (?)
Or maybe this is just me and users will understand the setting is an addition, and that the bug is fixed in default settings.

Reply:
Or maybe Disable sprite wobble should be removed entirely. Probably no one would have thought to add it had it not caused the sprite pane column fluctuation bug.

Reply:
Unless it was part of a reduce animations addon but I think think Scratch really has any others to begin with.

Reply:
> Or maybe Disable sprite wobble should be removed entirely. Probably no one would have thought to add it had the sprite pane column fluctuation bug not existed.

After receiving feedback about the removal of "fix footer", I unfortunately disagree :p

Reply:
Maybe a select setting:

Mode: `Fix glitch` / `Fix glitch and disable sprite wobble`